src/officehours_ui/static/css/main.css

```diff
@@ -200,9 +200,13 @@ p a {
 	padding: 1.25rem;
 }
 
+.text-danger {
+	color: #CC0000 !important;
```

Comment:
Without the `!important`, this wasn't applying, there was some `text-emphasis` Bootstrap asset that was taking precedence, I think.

Reply:
Is it possible to make this a more specific selector (adding selectors before it), or to move this elsewhere in the file to have it take precedence without `!important`? I try to avoid using it where possible, but sometimes it makes sense.

Reply:
Yeah, let me see what I can do.